Lib/sysconfig/__main__.py

```diff
@@ -157,6 +159,19 @@ def _print_config_dict(d, stream):
     print ("}", file=stream)
 
 
+def _get_pybuilddir():
+    pybuilddir = f'build/lib.{get_platform()}-{get_python_version()}'
+    if hasattr(sys, "gettotalrefcount"):
+        pybuilddir += '-pydebug'
+    return pybuilddir
```

Comment:
@FFY00 is there a reason you did this instead of:
```suggestion
    with open(f'{_PROJECT_BASE}/pybuilddir.txt', 'r', encoding='utf8') as f:
        return f.read()
```

Reply:
Oh sorry dumb question I see now this is the source of pybuilddir.txt.